app/src/main/java/com/edulectronics/tinycircuit/ui/CircuitActivity.java

```diff
@@ -17,7 +20,8 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
```

Comment:
De onCreate method wordt nu wel erg groot zou je deze nog op kunnen splitsen in kleinere methoden?